### ISSUE TITLE
Export cover letter as PDF

### ISSUE THREAD

**Issue by hdrury1** [elifesciences/elife-xpub#1497]
Currently, we convert the author's cover letter into a HTML file before export in the MECA package, and this is resulting in some unwanted behaviour in eJP. We should export the file in PDF format instead.

Note that, once we are no longer integrating with eJP, we will want to retain the HTML format so that we can display the cover letter within the application.

**Comment by hdrury1** [comment]
```
Andy   [1 hour ago]
Do you have an update on the lack of cover letter PDF? 
Unfortunately that's causing the Deputy Editors some problems. 
When the PDF isn't available, on their view, it looks like there isn't a cover letter at all.
```

**Comment by acollings** [comment]
@hdrury1 I'll be speaking with the editors later. Do you have an estimate for when this one might be ready? Thanks!

**Comment by hdrury1** [comment]
@acollings We're waiting on a code update from eJP which I've requested from Joel happen in time for testing on Friday.

**Comment by diversemix** [comment]
Change remains un-merged - this will be merged as part of #1583
